ui/pages/Project/selectors.js

```diff
@@ -51,16 +49,20 @@ export const getProject = createSelector(
   getProjectsByGuid, getProjectGuid, (projectsByGuid, currentProjectGuid) => projectsByGuid[currentProjectGuid],
 )
 
-const filterProjectEntities = (entities, currentProjectGuid) =>
-  Object.values(entities).filter(o => o.projectGuid === currentProjectGuid)
-
-export const getProjectFamilies = createSelector(getFamiliesByGuid, getProjectGuid, filterProjectEntities)
+const filterProjectEntities = (entities, currentProjectGuid) => Object.entries(entities).filter(
+  ([, o]) => o.projectGuid === currentProjectGuid,
+).reduce((acc, [k, o]) => {
+  return { ...acc, [k]: o }
+}, {})
 
-export const getProjectIndividuals = createSelector(getIndividualsByGuid, getProjectGuid, filterProjectEntities)
 
-export const getProjectDatasets = createSelector(getDatasetsByGuid, getProjectGuid, filterProjectEntities)
+export const getProjectFamiliesByGuid = createSelector(getFamiliesByGuid, getProjectGuid, filterProjectEntities)
+export const getProjectIndividualsByGuid = createSelector(getIndividualsByGuid, getProjectGuid, filterProjectEntities)
+export const getProjectSamplesByGuid = createSelector(getSamplesByGuid, getProjectGuid, filterProjectEntities)
 
-export const getProjectSamples = createSelector(getSamplesByGuid, getProjectGuid, filterProjectEntities)
+export const getProjectFamilies = createSelector(getProjectFamiliesByGuid, d => Object.values(d))
+export const getProjectIndividuals = createSelector(getProjectIndividualsByGuid, d => Object.values(d))
+export const getProjectSamples = createSelector(getProjectSamplesByGuid, d => Object.values(d))
```

Comment:
I think it would be better if we got rid of these and just used the `getProject<Thing>ByGuid` selectors everywhere for consistancy

Reply:
Sure, and probably also for getProjectSavedVariants?